CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+### Added
+- PRE and POST interaction commands
```

Comment:
Ah, sorry. I meant "this enables for example ..." like to give people idea what they can do with it (like it's in the config template).
I didn't want to do shairport-specific code. This here should be just a comment to give an idea.